docs/static/releasenotes.asciidoc

```diff
@@ -1,4 +1,291 @@
 [[releasenotes]]
 == Release Notes
-coming[9.0.0]
 
+* <<logstash-9-0-0-beta1,Logstash 9.0.0-beta1>>
+
+[[logstash-9-0-0-beta1]]
+=== Logstash 9.0.0-beta1 Release Notes
+
+==== Breaking changes
+* Default buffer type to 'heap' for 9.0 https://github.com/elastic/logstash/pull/16500[#16500]
```

Comment:
@karenzone WDYT if we forward to (or embed) breaking-changes-90.asciidoc for breaking changes? 
Bottom plugin obsoletion items wouldn't be necessarily here.

Reply:
Starting with 9.0, we'll be grouping our release notes with the rest of Elastic. We'll need to align with the approach that everybody else is using, and we'll be deciding that next week.  Stay tuned!

Reply:
We typically have a breaking changes section in the release notes with at least a brief description like in https://www.elastic.co/guide/en/logstash/current/logstash-8-0-0-alpha1.html.  The breaking changes aren't complete yet, so I'm not sure we should like away from release notes.